app/views/user_mailer/nsv_bsv_submission_rejected.html.erb

```diff
@@ -11,7 +11,11 @@
         </tr>
         <tr>
           <td class='email-sub-title'>
-            <p>Dear <%= @submission.name %>,</p>
+            <% if @submission.user_name %>
+              <p>Hi <%= @submission.user_name %>,</p>
```

Comment:
**Issue:** I believe the final draft has "Dear <name>," instead of "Hi <name>,"

Reply:
The collector services team opted to use `Hi` instead. See here: https://artsy.slack.com/archives/C01B2P6LJUU/p1671465006706159?thread_ts=1671463083.624579&cid=C01B2P6LJUU